src/components/mypage/ReviewModal/index.tsx

```diff
@@ -0,0 +1,135 @@
+import type { ChangeEventHandler, ReactElement } from 'react'
+import { useState } from 'react'
+import { Styled } from './styled'
+import type {
+  ReviewModalProps,
+  EvaluateState,
+  EvaluateData,
+  ReviewState
+} from './types'
+
+const evaluateData: EvaluateData = [
+  {
+    state: 'smile',
+    text: '좋아요'
+  },
+  {
+    state: 'meh',
+    text: '보통이에요'
+  },
+  {
+    state: 'sad',
+    text: '별로에요'
+  }
+]
+
+const ReviewModal = ({
+  isOpen = true,
+  onClose,
+  onClick,
+  nickName = '닉네임',
+  productName = '상품이름',
+  isReadMode = false,
+  readModeEvaluate = 'smile',
+  readModeReviewContent = '리뷰'
+}: ReviewModalProps): ReactElement => {
+  const [isClickReviewIcon, setIsClickReViewIcon] = useState<boolean>(false)
```

Comment:
```suggestion
  const [isClickReviewIcon, setIsClickReViewIcon] = useState<boolean>(isReadMode)
```
`isReadMode`를 기본값으로 주면 126라인에서 크로스 체크 안해도 괜찮을 것 같아요!

```ts
 <Styled.ReviewSendButton
        disabled={isReadMode ? false : !isClickReviewIcon}
        styleType={
          isClickReviewIcon ? 'solidPrimary' : 'solidDisabled'
        }
        onClick={handleClickSendReview}>
        {isReadMode ? '확인' : '후기 보내기'}
      </Styled.ReviewSendButton>
```

Reply:
해당 요구사항 선반영 하였으나, 
Read/Write 컴포넌트 로직 분리가 결정되면 resolve 하겠습니다!
[링크](https://github.com/price-offer/offer-fe/pull/108/commits/2f6a4f180e5cc19c7c09ed82b4805a718940c178#diff-4b8ae2495983bb2ac6378048f00a9113a977027de98963d259a27f5437cb9424)